accepted/2022/simplify-output-paths.md

```diff
@@ -0,0 +1,58 @@
+# Simplify .NET Output Paths
+
+We'd like to simplify the output paths for .NET 8.
+
+Currently, the default output path for a .NET project includes folders for the `TargetFramework` and the `RuntimeIdentifier` (if it is set).  Additionally, the publish output goes in a `publish` subfolder of the output folder.  The resulting paths are as follows:
+
+- `bin\<Configuration>\<TargetFramework>` - Build output with no `RuntimeIdentifier`
+- `bin\<Configuration>\<TargetFramework>\<RuntimeIdentifier>` - Build output with `RuntimeIdentifier`
+- `bin\<Configuration>\<TargetFramework>\publish` - Publish output with no `RuntimeIdentifier`
+- `bin\<Configuration>\<TargetFramework>\<RuntimeIdentifier>\publish` - Publish output with `RuntimeIdentifier`
+
+This is rather complicated an inconsistent, which can make for a poor first impression of the .NET platform.
+
+We'd like to try to improve the output path structure.  Desired qualities include:
+
+- Simple and consistent
+- Avoid cases where one output path is nested inside of another one
+- Avoid excessively deep directory heirarchies
+- Get rid of `obj` folder in project root
+
+## Proposed behavior
+
+Projects targeting .NET and higher will by default use a new output path format.  The output path will consist of the following 3 nested folders
+
+- `bin` - All output (including intermediate output, will go under this folder)
+- Output Type - Such as `build`, `publish`, `obj`, or `packages`
```

Comment:
Perhaps we could take this opportunity to rename `obj` to `intermediates` (or something to that effect), bringing the directory name in line with the property name (`IntermediateOutputPath`)?